tests/test_invenio_communities.py

```diff
@@ -284,34 +284,39 @@ def assert_header_links(response, ref, page, size):
 
 def test_communities_rest_get_details(app, db, communities):
     """Test the OAI-PMH Sets creation."""
+    (comm1, comm2, comm3) = communities
     with app.test_client() as client:
-        response = client.get('/api/communities/comm1')
+        response = client.get('/api/communities/{}'.format(str(comm1.id)))
```

Comment:
What about moving to a solution using PIDStore? We can take advantage of redirects, stub pages, etc. We could still keep the API using community name, but it would be a property with registered PID value.